spec/expression.dd

```diff
@@ -514,8 +514,9 @@ $(GNAME ShiftExpression):
         $(D >)$(D >)$(D >) is an unsigned right shift.
     )
 
-    $(P It's illegal to shift by the same or more bits than the size of the
-        quantity being shifted:
+    $(P If the quantity is shifted by the same or more bits than the size of the
+	quantity, it is implementation defined whether a diagnostic is issued
```

Comment:
I wanted to constrain it to issuing a diagnostic (which includes seg faulting and aborting) or producing a random integer value. I don't want to leave open "implementation defined" as including corrupting memory.